Automattic/sensei
# Auto-focus the course title field when creating a new course

Resolves https://github.com/Automattic/sensei/issues/7431

## Proposed Changes
* This will auto-focus the course title input when the course editor wizard is open.

## Testing Instructions


1. Go to Sensei LMS -> Courses -> New Course 
2. Make sure the "Course Title" input is auto-focused.

## Pre-Merge Checklist
